supportedBackends/google/pipelines/v2alpha1/src/test/scala/cromwell/backend/google/pipelines/v2alpha1/api/request/GetRequestHandlerSpec.scala

```diff
@@ -183,6 +183,89 @@ class GetRequestHandlerSpec extends FlatSpec with Matchers with TableDrivenPrope
+        |      "cromwell-workflow-id": "asdfasdf",
+        |      "wdl-call-alias": "validatecram",
+        |      "wdl-task-name": "validatesamfile"
+        |    },
+        |    "pipeline": {
+        |      "actions": [],
+        |      "environment": {},
+        |      "resources": {
+        |        "projectId": "",
+        |        "regions": [],
+        |        "virtualMachine": {
+        |          "accelerators": [],
+        |          "bootDiskSizeGb": 11,
+        |          "bootImage": "asdfasdf",
+        |          "cpuPlatform": "",
+        |          "disks": [
+        |            {
+        |              "name": "local-disk",
+        |              "sizeGb": 41,
+        |              "sourceImage": "",
+        |              "type": "pd-standard"
+        |            }
+        |          ],
+        |          "enableStackdriverMonitoring": false,
+        |          "labels": {
+        |            "cromwell-sub-workflow-name": "bamtocram",
+        |            "cromwell-workflow-id": "asdfasdf",
+        |            "goog-pipelines-worker": "true",
+        |            "wdl-call-alias": "validatecram",
+        |            "wdl-task-name": "validatesamfile"
+        |          },
+        |          "machineType": "custom-2-7168",
+        |          "network": {
+        |            "name": "",
+        |            "subnetwork": "",
+        |            "usePrivateAddress": false
+        |          },
+        |          "nvidiaDriverVersion": "",
+        |          "preemptible": false,
+        |          "serviceAccount": {
+        |            "email": "default",
+        |            "scopes": [
+        |              "https://www.googleapis.com/auth/genomics",
+        |              "https://www.googleapis.com/auth/compute",
+        |              "https://www.googleapis.com/auth/devstorage.full_control",
+        |              "https://www.googleapis.com/auth/cloudkms",
+        |              "https://www.googleapis.com/auth/userinfo.email",
+        |              "https://www.googleapis.com/auth/userinfo.profile",
+        |              "https://www.googleapis.com/auth/monitoring.write",
+        |              "https://www.googleapis.com/auth/cloud-platform"
+        |            ]
+        |          }
+        |        },
+        |        "zones": [
+        |          "us-central1-a",
+        |          "us-central1-b",
+        |          "us-east1-d",
+        |          "us-central1-c",
+        |          "us-central1-f",
+        |          "us-east1-c"
+        |        ]
+        |      },
+        |      "timeout": "604800s"
+        |    },
+        |    "startTime": "2019-08-18T12:04:39.192909594Z"
+        |  },
+        |  "name": "asdfasdf"
+        |}""".stripMargin, Failed(Status.ABORTED, None, Nil, List(ExecutionEvent("waiting for quota", OffsetDateTime.parse("2019-08-18T12:04:38.082650Z"),None)), Some("custom-2-7168"), None, None)
```

Comment:
NB this event list of 1 will clash with the event addition in #5220 - so one of us will have to update and re-test, depending on who merges first.